admin_manual/configuration_server/occ_command.rst

```diff
@@ -242,6 +243,14 @@ You can get the full filepath to an app::
 
 .. _background_jobs_selector_label:   
```

Comment:
This should stay just before the `Background jobs selector` title :)

Reply:
outch 🦌